Lighthouse fails to install on Windows due to axe-core@2.4.0

axe-core@2.4.0 is preventing `npm install` from completing successfully for lighthouse on Windows.
```
npm ERR! Windows_NT 10.0.15063
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install"
npm ERR! node v6.9.5
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE

npm ERR! axe-core@2.4.0 postinstall: `./bin/postinstall.sh`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the axe-core@2.4.0 postinstall script './bin/postinstall.sh'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the axe-core package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ./bin/postinstall.sh
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs axe-core
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls axe-core
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\dev\projects\lighthouse\npm-debug.log
```
Seems to be due to a shell script being used in the postinstall step, this is resolved in axe-core@2.4.1 which converts the shell script to node. See [axe-core #516](https://github.com/dequelabs/axe-core/issues/516).

--- a/package.json
+++ b/package.json
@@ -73,7 +73,7 @@
     "zone.js": "^0.7.3"
   },
   "dependencies": {
-    "axe-core": "2.4.0",
+    "axe-core": "2.4.1",
     "chrome-devtools-frontend": "1.0.422034",
     "devtools-timeline-model": "1.1.6",
     "jpeg-js": "0.1.2",

